pkg/commands/compute/setup/config_store.go

```diff
@@ -11,54 +11,52 @@ import (
 	"github.com/fastly/go-fastly/v8/fastly"
 )
 
-// Dictionaries represents the service state related to dictionaries defined
+// ConfigStores represents the service state related to config stores defined
 // within the fastly.toml [setup] configuration.
 //
 // NOTE: It implements the setup.Interface interface.
-type Dictionaries struct {
+type ConfigStores struct {
```

Comment:
Config stores are different to dictionaries, they have different naming rules and are also a shared resource like object stores and secret stores. Does change create config store resources and link them to the newly created fastly service?

Reply:
Good shout. I'll double-check in the morning.